docs/reference/tab-widgets/troubleshooting/disk/decrease-data-node-disk-usage.asciidoc

```diff
@@ -0,0 +1,141 @@
+In order to decrease the disk usage in your cluster without losing any data is to reduce the replicas of indices.
+
+NOTE: Reducing the replicas of an index can potentially reduce search performance and make the index less resilient to
+data loss. However, it can quickly give the cluster breathing room until a more permanent solution is in place.
+
+// tag::cloud[]
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment.
++
+
+NOTE: If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is
+{cloud}/ec-access-kibana.html[enable it first].
++
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Stack Management > Index Management**.
+
+. In the list of all your indices, click the `Replicas` column twice to sort the indices based on their number of
+replicas starting with the one that has the most. Go through the indices and pick one by one the index with the
+least importance and higher number of replicas.
```

Comment:
Shall we add a big warning here that the data redundancy and search throughput can be affected by this operation?

Reply:
Better safe than sorry :) I will repeat it.